apps/zero-runtime-vite-app/jest.config.js

```diff
@@ -0,0 +1,29 @@
+// Mandatory to map so that the imports translate to cjs files
+// eslint-disable-next-line import/no-unresolved
+const { createTheme } = require('@mui/material/node/styles');
+
+const theme = createTheme();
+// @TODO - Make this part of the main package
+// @ts-ignore
+theme.applyDarkStyles = function applyDarkStyles(obj) {
+  return {
+    // @TODO - Use custom stylis plugin as in docs/src/createEmotionCache.ts
+    // so that we don't need to use *
+    '* :where([data-mui-color-scheme="dark"]) &': obj,
+  };
+};
+
+/**
+ * @type {import('jest').Config}
+ */
+module.exports = {
+  transform: {
+    '\\.[jt]sx?': ['@mui/zero-jest', { theme }],
```

Comment:
Shouldn't we make the regexp accept .js files too?

Reply:
It does already. The regex covers js,ts,jsx and tsx